src/pages/profile/profile.js

```diff
@@ -8,6 +8,7 @@ export default function Profile() {
       <h1>Profile Working</h1>
       <ul>
         <li>
+          <h2>Hey, you are in the Queue ! You are the number ...</h2>
```

Comment:
I don't think you want to display the queue number at the profile page, something like this has to be at the start page